examples/app.py

```diff
@@ -31,10 +31,11 @@
 
     .. code-block:: console
 
-        cdvirtualenv src/invenio-marc21
         pip install -e .[all]
-        pip install invenio-theme==1.0.0a9
+        pip install -r requirements-devel.txt
```

Comment:
FWIW I like `examples/requirements-devel.txt` approach for consistency with the other Invenio packages.  And I would mention it with the "# optional" comment, so that the instructions would be valid also in the future when we'll release `dojson` on PyPI.